platform-dom/src/main/java/com/softicar/platform/dom/refresh/bus/DomRefreshBus.java

```diff
@@ -92,4 +90,35 @@ private void submitQueue() {
 		}
 		queue.clear();
 	}
+
+	private void convertDanglingNodeListeners() {
+
+		var bodyFinder = new DomParentNodeFinder<>(DomBody.class);
+		for (var listener: new ArrayList<>(listeners.keySet())) {
+			if (bodyFinder.findMostDistantParent(listener).isEmpty()) {
+				listeners.remove(listener);
+				listener//
+					.getDomDocument()
+					.getDeferredInitializationController()
+					.addDeferredInitializer(listener, new ReappendedNodeInitializer(listener));
+			}
+		}
+	}
+
+	private class ReappendedNodeInitializer implements INullaryVoidFunction {
+
+		private final IDomRefreshBusListener listener;
+
+		public ReappendedNodeInitializer(IDomRefreshBusListener listener) {
```

Comment:
the class name `ReappendedNodeInitializer` talks about a `node` while the parameter talks about a `listener`, IMO we should only use the term `listener` or `listeningNode` everywhere to avoid confusion

Reply:
granted